spring-cloud-gateway-core/src/main/java/org/springframework/cloud/gateway/support/ServerWebExchangeUtils.java

```diff
@@ -50,6 +50,7 @@
 	public static final String GATEWAY_PREDICATE_ROUTE_ATTR = qualify("gatewayPredicateRouteAttr");
 	public static final String WEIGHT_ATTR = qualify("routeWeight");
 	public static final String ORIGINAL_RESPONSE_CONTENT_TYPE_ATTR = "original_response_content_type";
+	public static final String HYSTRIX_EXECUTION_EXCEPTION = qualify("hystrixExecutionException");
```

Comment:
I would add this to the documentation as well

Reply:
:+1: and suffix with `_ATTR`.

Reply:
It's there - have mentioned you in a comment in the .adoc file